src/SwooleRequestHandlerRunner.php

```diff
@@ -162,11 +166,15 @@ public function run() : void
      */
     public function startServer(array $options = []) : void
     {
-        $swooleServer = $this->serverFactory->createSwooleServer($options);
-        $swooleServer->on('start', [$this, 'onStart']);
-        $swooleServer->on('workerstart', [$this, 'onWorkerStart']);
-        $swooleServer->on('request', [$this, 'onRequest']);
-        $swooleServer->start();
+        if ($this->isRunning()) {
```

Comment:
There was previously no check for whether the server was already running or not. Was this intentional for some reason that I've missed?

Reply:
Not intentional. We covered it for reload and stop operations, but missed it for start operations. Good catch!